CoffeeChat/Cells/OnboardingTableViewCell.swift

```diff
@@ -53,13 +57,17 @@ class OnboardingTableViewCell: UITableViewCell {
         backdropView.layer.shadowOpacity = 0.1
         backdropView.layer.shadowRadius = 2
         contentView.addSubview(backdropView)
+
+        categoriesLabel.textColor = .greenGray
+        categoriesLabel.font = ._12CircularStdBook
+        contentView.addSubview(categoriesLabel)
     }
 
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
 
-    private func setupConstraints(showingInterests: Bool) {
+    private func setupConstraints() {
         let imageSize = CGSize(width: 22, height: 22)
         let sidePadding: CGFloat = 12
```

Comment:
I think since side padding is just used once, it doesn't need to be its own variable?